ext/css/action-popup.css

```diff
@@ -497,6 +387,104 @@ select.profile-select {
     background-color: var(--warning-color);
 }
 
+/* Action Containers */
+
+.action-icon:hover {
+    cursor: pointer;
+}
+
+
+.action-icon {
+    flex: 0 0 auto;
+    align-self: center;
+}
+.action-icon>.icon {
+    display: block;
+    background-color: var(--button-default-icon-color);
+    width: 1.5em;
+    height: 1.5em;
+}
+
+.action-item {
+    position: relative;
+    padding: .5em;
+}
+.action-item:not([hidden]) {
+    display: block;
+}
+.action-item-outer {
+    display: block;
+    width: 100%;
+}
+.action-item-inner {
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: flex-end;
+    align-content: stretch;
+    width: 100%;
+}
+
+.action-item-left {
+    padding: var(--action-group-inner-vertical-padding) var(--action-group-inner-horizontal-padding-half) var(--action-group-inner-vertical-padding) var(--action-group-inner-horizontal-padding);
+    flex: 1 1 auto;
+    align-self: center;
+    position: relative;
+}
+.action-item-center {
+    padding: var(--action-group-inner-vertical-padding) var(--action-group-inner-horizontal-padding);
+    flex: 1 1 auto;
+    align-self: center;
+    display: flex;
+    flex-flow: column nowrap;
+    align-items: center;
+    justify-content: center;
+}
+.action-item-right {
+    padding: var(--action-group-inner-vertical-padding) var(--action-group-inner-horizontal-padding) var(--action-group-inner-vertical-padding) var(--action-group-inner-horizontal-padding-half);
+    flex: 0 1 auto;
+    align-self: stretch;
+    max-height: var(--action-group-right-max-height);
+    display: flex;
+    flex-flow: row nowrap;
+    align-items: center;
+    align-content: center;
+    justify-content: flex-end;
+    padding-left: .5em;
+}
+
+.action-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 10px;
+}
+
+.action-container button {
+    flex: 0 0 auto;
+}
+
+/* Misc */
+
+h2 {
```

Comment:
This is pretty broad. Can we scope it down more? Maybe we can use a span instead of an `h2` tag?